src/global.c

```diff
@@ -273,8 +273,9 @@ static void init_once(void)
 {
 	if ((init_error = git_mutex_init(&git__mwindow_mutex)) != 0)
 		return;
-	pthread_key_create(&_tls_key, &cb__free_status);
 
+	if (!_tls_key)
+		pthread_key_create(&_tls_key, &cb__free_status);
```

Comment:
When is `_tls_key` ever non-NULL here?  Isn't `pthread_once` handling this for us?